std/experimental/ndslice/internal.d

```diff
@@ -1,3 +1,5 @@
+/// @@@DEPRECATED_2017-04@@@
+deprecated("Please use mir-algorithm DUB package: http://github.com/libmir/mir-algorithm")
```

Comment:
@9il I just realized that the repo doesn't exist yet. Do you plan to create it or did you mean to link to https://github.com/libmir/mir-std?